packages/nodes-base/nodes/Jira/IssueDescription.ts

```diff
@@ -550,52 +550,29 @@ export const issueFields = [
 				name: 'expand',
 				type: 'string',
 				default: '',
-				description: `Use expand to include additional information about the issues in the response.<br/>
-				This parameter accepts a comma-separated list. Expand options include:<br/>
-				renderedFields Returns field values rendered in HTML format.<br/>
-				names Returns the display name of each field.<br/>
-				schema Returns the schema describing a field type.<br/>
-				transitions Returns all possible transitions for the issue.<br/>
-				editmeta Returns information about how each field can be edited.<br/>
-				changelog Returns a list of recent updates to an issue, sorted by date, starting from the most recent.<br/>
-				versionedRepresentations Returns a JSON array for each version of a field's value, with the highest number
+				description: `Use expand to include additional information about the issues in the response. This parameter accepts a comma-separated list. Expand options include: <code>renderedFields</code> Returns field values rendered in HTML format. <code>names</code> Returns the display name of each field. <code>schema</code> Returns the schema describing a field type. <code>transitions</code> Returns all possible transitions for the issue. <code>editmeta</code> Returns information about how each field can be edited. <code>changelog</code> Returns a list of recent updates to an issue, sorted by date, starting from the most recent. <code>versionedRepresentations</code> Returns a JSON array for each version of a field's value, with the highest number
```

Comment:
not really readable. we need to break this.
<img width="423" alt="Screen Shot 2021-11-24 at 12 49 38 PM" src="https://user-images.githubusercontent.com/4711238/143233253-cb8e93d0-2643-4f15-a157-8518423454b1.png">